### ISSUE TITLE
Add `rdma_mpi` to test suite

### ISSUE THREAD

**Issue by sbryngelson** [MFlowCode/MFC#598]
Add `rdma_mpi = 'T'` to test suite. This only gets turned on in OpenACC (GPU) mode, which only exists on the Phoenix GPU runner and the Frontier runner, both of which support it.

Note: There needs to be a way to skip this test if the compiler does not support it (in non-CI cases).

**Comment by sbryngelson** [comment]
One option: add `--rdma` to `./mfc.sh test` as an optional flag that adds RDMA tests and is used in CI, but is off by default.

**Comment by aricer123** [comment]
"Add rdma_mpi = 'T' to test suite" I'm a little confused by what this means. 
So far, I have created a flag for test that passes to run provided that we are running on a gpu/gpu mode.
Is there a feature already that runs RDMA tests if an --rdma flag is set in run? What work has been done with this feature so far and what more do I need to implement?

**Comment by sbryngelson** [comment]
> Is there a feature already that runs RDMA tests if an --rdma flag is set in run?

No, it is set in the `case.py` file. This (currently nonexistent) flag, or something like it, would have to be added so turning on this feature could also be controlled from the command line.
